src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,51 @@
+.clock {
+  position: absolute;
+  @include round ($clock-size);
+  background-color: $clock-color;
+
+  &__center {
+    position: absolute;
+    z-index: 1;
```

Comment:
Don't need to set this if you place `.clock__center` after all clock hands in HTML. It will be on top of them.